proto/proto-backplane-grpc/src/main/proto/deephaven/proto/table.proto

```diff
@@ -381,7 +381,10 @@ message UpdateByWindowScale {
   }
   message UpdateByWindowTime {
     string column = 1;
-    sint64 period_nanos = 2 [jstype=JS_STRING];
+    oneof window {
+      sint64 nanos = 2 [jstype = JS_STRING];
+      string duration_string = 3;
```

Comment:
Do we want / need to add documentation here about this string format? `DateTimeUtils.parseDurationNanos`?

Reply:
I updated the function signatures to reflect that a time string is also allowed.  The current description is fairly vague however:
`a time interval string, e.g. "PT00:00:00.001"`

Reply:
The overall docs should cover the formatting of the string, so it shouldn't be as critical to have it in every location.  Also having it in every location may be more brittle, if we support more formats in the future.